packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.service.js

```diff
@@ -0,0 +1,305 @@
+import includes from 'lodash/includes';
+import find from 'lodash/find';
+
+export default class HostingCdnSharedService {
+  /* @ngInject */
+  constructor($http, OvhHttp) {
+    this.$http = $http;
+    this.OvhHttp = OvhHttp;
```

Comment:
Why use both ? Only one http client seems needed

Reply:
initially i used only OvhHttp but for unkown raison two end points doesnt work with it.
so i use $http.